sympy/vector/coordsysrect.py

```diff
@@ -243,6 +337,9 @@ def base_scalars(self):
     def lame_coefficients(self):
         return self._h1, self._h2, self._h3
 
+    def transformation_equations(self):
+        return self._transformation_eqs
```

Comment:
Maybe this should return `self._transformations_eqs[:]`, so we make sure that it returns a copy, and not a link to the internal variable.